scripts/create-local-tag-for-release

```diff
@@ -22,7 +25,7 @@ HELP=$(cat << 'EOM'
     -h      help
 
   Examples:
-    create-local-tag-for-release -v v1.0.0       Create local tag for new version v1.0.0
+    create-local-tag-for-release -v v1.0.0      Create local tag for new version v1.0.0
     create-local-tag-for-release -i             Create local tag for new version by incrementing minor version only (previous tag=1.0.0, new tag=1.1.0)
```

Comment:
tags should have the prepended `v`  `tag=v1.0.0, new tag=v1.1.0`

Reply:
good catch.